pvlib/tests/test_snow.py

```diff
@@ -95,3 +95,32 @@ def test_dc_loss_nrel():
     expected = pd.Series([1, 1, .5, .625, .25, .5, 0])
     actual = snow.dc_loss_nrel(snow_coverage, num_strings)
     assert_series_equal(expected, actual)
+
+
+def test__townsend_Se():
+    S = np.array([10, 10, 5, 1, 0, 0, 0, 0, 0, 0, 5, 10])
+    N = np.array([2, 2, 1, 0, 0, 0, 0, 0, 0, 0, 2, 3])
+    expected = np.array([7.5, 7.5, 5, 0, 0, 0, 0, 0, 0, 0, 3.75, 6.66666667])
+    actual = snow._townsend_Se(S, N)
+    np.testing.assert_allclose(expected, actual, rtol=1e-07)
+
+
+def test_loss_townsend():
```

Comment:
Question for @cwhanse: referring to https://github.com/pvlib/pvlib-python/pull/1393#issuecomment-1022336293, should we have a policy of always testing both array and Series for `array-like` parameters?

Reply:
> should we have a policy of always testing both array and Series for array-like parameters?

I think so

Reply:
I'd say it's good practice but importance and extent depends on the model and implementation. I'm usually more concerned that we test for compatibility with scalars in "float, array, series" situations because I think it's easier for regressions to slip in with scalars (e.g. by introducing masking).